hs_geo_raster_resource/migrations/0001_initial.py

```diff
@@ -1,99 +0,0 @@
-                ('discoverable', models.BooleanField(default=True, help_text=b'If this is true, it will turn up in searches.')),
-                ('published_and_frozen', models.BooleanField(default=False, help_text=b'Once this is true, no changes can be made to the resource')),
-                ('content', models.TextField()),
-                ('short_id', models.CharField(default=hs_core.models.short_id, max_length=32, db_index=True)),
-                ('doi', models.CharField(help_text=b"Permanent identifier. Never changes once it's been set.", max_length=1024, null=True, db_index=True, blank=True)),
-                ('object_id', models.PositiveIntegerField(null=True, blank=True)),
-                ('content_type', models.ForeignKey(blank=True, to='contenttypes.ContentType', null=True)),
-                ('creator', models.ForeignKey(related_name='creator_of_hs_geo_raster_resource_rasterresource', to=settings.AUTH_USER_MODEL, help_text=b'This is the person who first uploaded the resource')),
-                ('edit_groups', models.ManyToManyField(help_text=b'This is the set of Hydroshare Groups who can edit the resource', related_name='group_editable_hs_geo_raster_resource_rasterresource', null=True, to='auth.Group', blank=True)),
-                ('edit_users', models.ManyToManyField(help_text=b'This is the set of Hydroshare Users who can edit the resource', related_name='user_editable_hs_geo_raster_resource_rasterresource', null=True, to=settings.AUTH_USER_MODEL, blank=True)),
-                ('last_changed_by', models.ForeignKey(related_name='last_changed_hs_geo_raster_resource_rasterresource', to=settings.AUTH_USER_MODEL, help_text=b'The person who last changed the resource', null=True)),
-                ('owners', models.ManyToManyField(help_text=b'The person who uploaded the resource', related_name='owns_hs_geo_raster_resource_rasterresource', to=settings.AUTH_USER_MODEL)),
-                ('user', models.ForeignKey(related_name='rasterresources', verbose_name='Author', to=settings.AUTH_USER_MODEL)),
-                ('view_groups', models.ManyToManyField(help_text=b'This is the set of Hydroshare Groups who can view the resource', related_name='group_viewable_hs_geo_raster_resource_rasterresource', null=True, to='auth.Group', blank=True)),
-                ('view_users', models.ManyToManyField(help_text=b'This is the set of Hydroshare Users who can view the resource', related_name='user_viewable_hs_geo_raster_resource_rasterresource', null=True, to=settings.AUTH_USER_MODEL, blank=True)),
-            ],
-            options={
-                'ordering': ('_order',),
-                'verbose_name': 'Geographic Raster Resource',
-            },
-            bases=('pages.page', models.Model),
-        ),
-        migrations.AlterUniqueTogether(
-            name='cellinformation',
-            unique_together=set([('content_type', 'object_id')]),
-        ),
-    ]
```

Comment:
Existing migrations that have already been run on other machines cannot be deleted.